packages/components/switch/src/SwitchInput.tsx

```diff
@@ -82,6 +95,13 @@ export const Input = React.forwardRef<HTMLButtonElement, InputProps>(
         checked={checked}
         defaultChecked={defaultChecked}
         onCheckedChange={handleCheckedChange}
+        id={id || controlledId}
+        name={name || controlledName}
+        required={required || isRequired}
+        aria-required={required || isRequired}
+        aria-invalid={isInvalid}
+        aria-labelledby={labelId}
```

Comment:
If there is already a label this is not needed 🤔 We can discuss it tomorrow if you want

Reply:
Based on the possibility to use Switch within a group, and/or following the GPDR settings pattern we saw together, I guess we should leave it as is, wdyt?

Reply:
Or should we omit this one for now, and wait for Design team to eventually add it?

Reply:
If there are `children` that means that a label is set so `aria-labelledby` is not needed.
```
aria-labelledby={children ? undefined : labelId}
```

Reply:
WDYT ?

Reply:
BTW I think that an story for a visually hidden label is needed but we can create another issue for that

Reply:
After doing some tests, it seems there is no accessible name available without adding this `aria-labelledby` :thinking: The `name` attribute doesn't seem to be added (Radix implementation) here, and the `role"switch"` doesn't match with the label (`Switch` child), as they're siblings.
I'm not sure if we should update the DOM structure, or if we should stick with this `aria-labelledby` anyway... wdyt?